### PR TITLE
Bump ld-find-code-refs-github-action version to 1.3.0

### PR DESCRIPTION
**Pull request by atrakh**

### CODE REVIEW

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -1,14 +0,0 @@
-on: push
```
Comment (atrakh) [inline .github/workflows/action.yml]: this snippet is already in the readme- let's remove it from the directory structure so it doesn't actually run as an action

Comment (vlymar) [reply .github/workflows/action.yml]: hmm yeah why did we have this here. wonder if i was just using it to test things

Comment (bwoskow-ld) [reply .github/workflows/action.yml]: I added this when trying to get the action working for the launch, and there was a lot of confusion at the time of what file we needed to add and how to specify it....

**Comment by sumuis** [comment]
...

**Pull request merged**